tuf-spec.md

```diff
@@ -1358,8 +1356,6 @@ non-volatile storage as FILENAME.EXT.
     released versions of root metadata files should always be provided
     so that outdated clients can update to the latest available root.
 
-    For more details on how this would apply on a community repository, please
-    see https://github.com/theupdateframework/pep-on-pypi-with-tuf#producing-consistent-snapshots
```

Comment:
As mentioned in the PR description, I'm happy to update this link (instead of removing it). I suggest to point to the official PEP, ie. https://www.python.org/dev/peps/pep-0458/#consistent-snapshots, which should soon have integrated our recent changes made in
https://github.com/secure-systems-lab/peps/blob/master/pep-0458.txt.